main/src/main/java/cgeo/geocaching/utils/ShareUtils.java

```diff
@@ -78,10 +79,18 @@ private static void shareAsEmail(final Context context, final String subject, fi
 
     private static void shareInternal(final Context context, final Intent intent, @StringRes final int titleResourceId) {
```

Comment:
`shareInternal` gets used for many different types of share intents, including e.g. file share. Maybe make it a text share only feature?

Reply:
Ok, restricted to text input